tools/versioner/cmd/root.go

```diff
@@ -128,19 +137,38 @@ func forSideBySideRelease(stage string, mod modinfo.Provider) (string, error) {
 	}
 	// must close file before renaming directory
 	file.Close()
+	var tag string
+	if tag, err = calculateModuleTag(nil, mod); err != nil {
+		return "", fmt.Errorf("failed to calculate module tag: %v", err)
+	}
+	if err := updateVersion(stage, tag); err != nil {
+		return "", fmt.Errorf("failed to update version.go: %v", err)
+	}
 	// move staging to new LMV directory
 	if err = os.Rename(stage, mod.DestDir()); err != nil {
 		return "", fmt.Errorf("failed to rename '%s' to '%s': %v", stage, mod.DestDir(), err)
 	}
-	var tag string
-	if tag, err = calculateModuleTag(nil, mod); err != nil {
-		return "", fmt.Errorf("failed to calculate module tag: %v", err)
+	// format code
+	if err = formatCode(mod.DestDir()); err != nil {
+		return "", fmt.Errorf("failed to format code: %v", err)
 	}
 	return tag, nil
 }
 
 // releases the module as an in-place update (minor or patch)
 func forInplaceUpdate(lmv, stage string, mod modinfo.Provider) (string, error) {
+	vprintln("This is a inplace update")
+	goMod := filepath.Join(stage, "go.mod")
+	file, err := os.OpenFile(goMod, os.O_RDWR, 0666)
+	if err != nil {
+		return "", fmt.Errorf("failed to open for read '%s': %v", goMod, err)
+	}
+	ver := modinfo.FindVersionSuffix(mod.DestDir())
+	if err = updateGoModVer(file, ver); err != nil {
+		file.Close()
+		return "", fmt.Errorf("failed to update go.mod file: %v", err)
+	}
+	file.Close()
```

Comment:
Inplaced update also need to update go.mod file, since it may be a update to v2 version.
I suppose the go.mod file generated by autorest will always be as `v1`, am I right? @jhendrixMSFT

Reply:
You are correct autorest always generates go.mod as v1 since it has no version information available to it.
Func `forInplaceUpdate()` does not need to modify the go.mod file, see the comment for that func.  `forSideBySideRelease()` is what updates go.mod for v2+

Reply:
You mean if there is already a major version of v2, say v2.0.0, and the content in stage folder do not have breaking changes comparing to v2, the tool will use `forInplaceUpdate()`. 
The test on scenarioe will fail if `forInplaceUpdate` do not modify `go.mod` file.

Reply:
ScenarioE is an in-place update so the version should end up being v2.1.0.  If the test is now failing then a regression has been introduced someplace.  Or maybe I misunderstand?

Reply:
Will the inplace update suppose to move every file other than `go.mod` to the old folder? Since autorest always generate `go.mod` as `v1`, I suppose in inplace update, we should not move the `go.mod` in stage folder to the corresponding major version folder. 
I will investigate this more carefully.

Reply:
Good point, I don't recall if this case is handled properly; it might not be.  I agree though we shouldn't move go.mod in this scenario so we can just delete it from the stage directory.